### DISCUSSION TITLE
Packet proxy example

### DISCUSSION THREAD

**Discussion by maxsupermanhd** [Tnze/go-mc#165] [Q&A] [answered]
Is there any example how to proxy client connection? I want to make terrain grabber

**Comment by Tnze** [comment]
https://github.com/go-mc/pernod

**Reply by maxsupermanhd** [reply]
It does not seem up to date and does it work?

**Reply by maxsupermanhd** [reply]
Bump. Looks like it is severely out of date, I can not figure how PlayerList shoud work, I guess there were changes that broke player storage... Any fixes? I just want to grab chunk data packet, not even modify it...

**Comment by maxsupermanhd** [comment]
Can someone help me implement a "/join <address>" command to a server that will direct player to address specified and capture terrain in between? Since Microsoft migration I think you also have to specify your token too but I assume it is prepared and sitting in a configuration file.

**Comment by maxsupermanhd** [comment, marked as answer]
So after a LOT of tinkering around I made something more or less working: https://github.com/maxsupermanhd/WebChunk/blob/master/proxy/proxy.go
